usr/lib/common/lock_sess_mgr.c

```diff
@@ -586,12 +586,14 @@ CK_RV session_mgr_logout_all(STDLL_TokData_t *tokdata)
 
 //
 //
-CK_RV session_mgr_get_op_state(SESSION *sess,
+CK_RV session_mgr_get_op_state(STDLL_TokData_t *tokdata, SESSION *sess,
                                CK_BBOOL length_only,
                                CK_BYTE *data, CK_ULONG *data_len)
 {
     OP_STATE_DATA *op_data = NULL;
-    CK_ULONG op_data_len = 0;
+    CK_ULONG max_data_len = *data_len;
+    CK_ULONG op_data_len;
```

Comment:
Can it happen that other input parms than sess, e.g. data_len, are NULL?

Reply:
This is already checked in SC_GetOperationState.